app/views/case_contacts/index.html.erb

```diff
@@ -15,139 +15,143 @@
-      <div class="h4 btn-lg">Filter by</div>
-      <button href="#"
-              class="btn btn-lg btn-link"
-              type="button"
-              data-bs-toggle="collapse"
-              data-bs-target="#filter-card-body"
-              aria-expanded="false"
-              aria-controls="filter-card-body">
-        <div class="h4">Show / Hide</div>
-      </button>
-    </div>
 
-    <div class="card-content mb-10 ml-20 collapse" id="filter-card-body">
-      <div class="row">
-        <div class="col-12">
-          <h3 class="mt-10"><label>Draft status</label></h3>
-        </div>
-        <div class="col-md-4">
+  <div class="card-style">
+    <div class="card-content">
+      <div class="d-flex justify-content-between align-items-center">
+        <div class="h4 btn-lg">Filter by</div>
+        <button href="#"
+                class="btn btn-lg btn-link"
+                type="button"
+                data-bs-toggle="collapse"
+                data-bs-target="#filter-card-body"
+                aria-expanded="<%= expand_filters? %>>"
+                aria-controls="filter-card-body">
+          <div class="h4">Expand / Hide</div>
+        </button>
+      </div>
+      <div class="d-flex flex-wrap justify-content-between align-items-end gap-3 mb-4">
+        <div class="d-flex flex-column gap-3">
+          <div class="select-style-1 mb-0">
+            <%= f.label :sorted_by %>
+            <div class="select-position">
+              <%= f.select(:sorted_by, @filterrific.select_options[:sorted_by], {}, {class: ""}) %>
+            </div>
+          </div>
           <div class="form-check checkbox-style">
             <%= f.check_box :no_drafts, class: "form-check-input case-contact-contact-type" %>
             <label class="form-check-label" for="filterrific_no_drafts">Hide drafts</label>
           </div>
         </div>
+        <div>
+          <%= button_tag( :class=> "btn-sm main-btn dark-btn") do %>
+            <i class="lni lni-funnel mr-10"></i> Filter
+          <% end %>
+          <%= link_to("Reset filters", reset_filterrific_url, class: "btn-sm main-btn dark-btn-outline btn-hove") %>
+        </div>
       </div>
-        <div class="row">
-          <div class="col-12">
-            <h3 class="mt-10"><label>Date of contact</label></h3>
-          </div>
-          <div class="col-sm-6 input-style-1">
-            <%= f.label "Starting from", for: "filterrific_occurred_starting_at" %>
-            <%= f.date_field(:occurred_starting_at, class: "") %>
-          </div>
-          <div class="col-sm-6 input-style-1">
-            <%= f.label "Ending at", for: "filterrific_occurred_ending_at" %>
-            <%= f.date_field(:occurred_ending_at, class: "") %>
-          </div>
+    </div>
+
+    <% collapse_class = expand_filters? ? "collapse show" : "collapse" %>
+    <div class="card-content mb-10 ml-20 <%= collapse_class %>" id="filter-card-body">
```

Comment:
If you wanted to be fancy you could do:

```
<div class="<%= token_list("card-content mb-10 ml-20 collapse", {show: expand_filters?}) %>" id="filter-card-body">
```
https://apidock.com/rails/ActionView/Helpers/TagHelper/token_list